src/main/java/com/shopify/model/ShopifyCustomCollectionCreationRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.shopify.model;
+
+public class ShopifyCustomCollectionCreationRequest {
+
+	private final ShopifyCustomCollection request;
+
+
+	public static interface InitialValueStep {
+		HandleStep withTitle(final String title);
+	}
+
+	public static interface HandleStep {
+		BuildStep withHandle(final String handle);
+	}
+
+	public static interface BuildStep {
+		ShopifyCustomCollectionCreationRequest build();
+	}
+
+
+	public static InitialValueStep newBuilder() {
+		return new Steps();
+	}
+
+	public ShopifyCustomCollection getRequest() {
+		return request;
+	}
+
+	private ShopifyCustomCollectionCreationRequest(final ShopifyCustomCollection request) {
+		this.request = request;
+	}
+
+	private static class Steps implements InitialValueStep, HandleStep, BuildStep {
+
+		private final ShopifyCustomCollection request = new ShopifyCustomCollection();
+
+		@Override
+		public ShopifyCustomCollectionCreationRequest build() {
+			return new ShopifyCustomCollectionCreationRequest(request);
+		}
+
+		@Override
+		public HandleStep withTitle(final String title) {
```

Comment:
Is there a reason you have excluded some of the writable fields from this step builder? 

body_html
template_suffix
published
sort_order

I think since you have them in the model, you can just add steps to be able to create a custom collection with them